tests/graphql/01-workflow.t

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) 2008-2019 NIWA & British Crown (Met Office) & Contributors.
+# 
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#-------------------------------------------------------------------------------
+# Test suite graphql interface
+. $(dirname $0)/test_header
+#-------------------------------------------------------------------------------
+set_test_number 4
+#-------------------------------------------------------------------------------
+install_suite "${TEST_NAME_BASE}" "${TEST_NAME_BASE}"
+
+# run suite
+run_ok "${TEST_NAME_BASE}-run" cylc run "${SUITE_NAME}"
+
+# query suite
+TEST_NAME="${TEST_NAME_BASE}-workflows"
+run_graphql_ok "${TEST_NAME}" "${SUITE_NAME}" '
+    query {
+        workflows {
```

Comment:
Yes, BTW the reason for the wrapper is that this:

```json
{
    "request_string": "
        query {
            workflows {
                name
            }
        }
    "
}
```

is actually invalid JSON (because of the newlines)

Reply:
Yes, it's just for presentation, otherwise it could all be on one line.